invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/GenerateForm.js

```diff
@@ -134,7 +135,7 @@ const mapFormFields = (obj, parentField, isCreate, formFieldsConfig, dropDumpOnl
       );
     }
 
-    if (fieldSchema.type === "object") {
+    if (fieldSchema.type === "object" && fieldSchema.metadata?.type !== "json") {
```

Comment:
minor: wouldn't it be better to explicitly have a clause for the `"json"` type? Or are there cases where this type is "inferred" somewhere and we have to exclude it explicitly here?

Reply:
I set it as json in the metadata.type in the schema otherwise it is interpreted as an object, hence why I had to add this check here. Could have made a separate for just json but it is just rendering a TextField for now like strings so I put it there for now as we are planning on adding support for json/code later I assume https://github.com/inveniosoftware/invenio-administration/blob/636066fa38e7d030758496d4604a8948a7ab2bcc/invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/GenerateForm.js#L175 
```
    if (
      (fieldSchema.type === "string" && rows) ||
      fieldSchema.metadata?.type === "json"
    ) {
      return (
        <TextArea
          key={fieldProps.fieldPath}
          fieldPath={fieldProps.fieldPath}
          rows={rows}
          {...fieldProps}
        />
      );
    }
```